third_party/fiat/curve25519_64.h

```diff
@@ -11,8 +11,9 @@
 #include <stdint.h>
 typedef unsigned char fiat_25519_uint1;
 typedef signed char fiat_25519_int1;
-typedef signed __int128 fiat_25519_int128;
-typedef unsigned __int128 fiat_25519_uint128;
+// Pedantic warnings can be disabled by adding prefix __extension__.
```

Comment:
So what's the warning here? I don't like disabling warnings, this file and p256_64.h should be included only when `__int128` types are supported.

Reply:
Explained in PR description.

ISO C does not support ‘__int128’ types [-Werror=pedantic].
The error was in the header file of the third-party lib fiat. The warning is suppressed by __extension__.

I plan to revert this change since `pedantic` will trigger c99 related warning. I don't see any benefits of making this fix since I don't plan to enable `pedantic` (until c99 requirements are clarified). What do you think?

Reply:
Merged this PR. Will move this thread discussion to slack.